otx/algorithms/action/adapters/mmaction/data/dataset.py

```diff
@@ -15,13 +15,14 @@
 # and limitations under the License.
 
 from copy import copy
-from typing import Any, Dict, List, Sequence
+from typing import List, Sequence
```

Comment:
How about changing this filename to `cls_dataset.py`?

Reply:
Changed!